packages/@azure/servicebus/data-plane/lib/session/messageSession.ts

```diff
@@ -587,6 +587,10 @@ export class MessageSession extends LinkEntity {
                 translatedError
               );
               this._notifyError(translatedError);
+            } finally {
+              if (this._receiver) {
+                this._receiver!.addCredit(1);
```

Comment:
We had disabled the `maxConcurrentCalls` and `maxConcurrentCallsPerSession` options to be customized by the user as we didn't understand them then. As part of https://github.com/Azure/azure-service-bus-node/issues/179, we were to bring them back.

In which case, shouldn't we be using `this.maxConcurrentCallsPerSession` here instead of hard-coding 1?